src/compute-types/src/dyncfgs.rs

```diff
@@ -122,6 +122,14 @@ pub const COPY_TO_S3_MULTIPART_PART_SIZE_BYTES: Config<usize> = Config::new(
     "The size of each part in a multipart upload to S3.",
 );
 
+/// Sets the max lifetime for replicas configured as an offset to the replica start time.
+/// Diffs generated at timestamps beyond the expiration time are dropped.
+pub const COMPUTE_REPLICA_EXPIRATION: Config<Duration> = Config::new(
+    "compute_replica_expiration",
+    Duration::from_secs(22 * 24 * 60 * 60), // 22 days
```

Comment:
Let's try to make this more readable: Define a constant `DURATION_DAY` as 86400 seconds, and multiply this with 22.